docs/source/notebookFiles/def2gds.py

```diff
@@ -0,0 +1,155 @@
+import copy
```

Comment:
These files are already in the sensor's directory. Why don't you point to them?
Same for the gds files above.